platform/viewer/src/connectedComponents/Viewer.js

```diff
@@ -557,7 +578,25 @@ const _mapStudiesToThumbnails = function(studies) {
 
       const hasWarnings = _checkForSeriesInconsistencesWarnings(displaySet, studies);
 
+      let active = false;
+      if (displaySet.Modality !== 'SEG' && displaySet.Modality !== 'RTSTRUCT') {
```

Comment:
We sometimes have RTDOSE and in the future, we could possibly support new modalities... I'm ok for now to have these hardcoded checks but we may want to add a TODO to improve it. 

Another thing, can you extract this whole block of code into its own function + add jsdocs?

Reply:
done see https://github.com/OHIF/Viewers/pull/2397/files#diff-4587d3942badf11b333cbd33f64c30d642840db46b853e146a5a514a92f0176eR548-R593